io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/GMLConstants.java

```diff
@@ -38,4 +38,9 @@ public interface GMLConstants {
 	 */
 	public static final String NS_GML_32 = "http://www.opengis.net/gml/3.2";
 
+	/**
+	 * The GML 3.2 namespace
```

Comment:
The constant name and its documentation should reflect that this is only part of a concrete WFS namespace (In other cases I think `prefix` or `core` are used in the name to symbolize this).